crates/ruma-events/src/enums.rs

```diff
@@ -8,6 +8,20 @@ use serde_json::value::RawValue as RawJsonValue;
 
 use super::room::encrypted;
 
+/// [Stripped state] event types which the spec recommends that servers should send so that clients
+/// can identify rooms
+///
+/// [Stripped state]: https://spec.matrix.org/latest/client-server-api/#stripped-state
+pub const RECOMMENDED_STRIPPED_STATE_EVENT_TYPES: [StateEventType; 7] = [
```

Comment:
I believe that it should be a slice, otherwise this is a fixed-size type and adding or removing items in the list would be a breaking change.